README.md

```diff
@@ -132,6 +132,98 @@ In order to support atomify turtles all the way down, you can also specify your
 
 For detailed information on configuring Rework plugins in package.json see the [relevant section](https://github.com/Techwraith/atomify-css#packagejson-config) of the atomify-css README.
 
+## Mini Tutorial
+### How to build a re-usable component
+(__Warning:__ _The following example is only interesting from a technical and not at all from an esthetical perspective)_
```

Comment:
1. utility classes are installed via a component who's job it is to provide them. Other components require that component.
2. We don't really use the `.is` classes. You're right, we should remove that from the styleguide.

re: decedents: every component has it's own namespace. No exceptions. This means that, you'd do `.myComponent-header` and `.myComponent-footer`. If header was a component, it would have `myHeader-logo` etc…